folium/folium.py

```diff
@@ -16,6 +16,8 @@
 import json
 from uuid import uuid4
 
+import imageio
```

Comment:
I think its best to import this inside `def image_overlay()` to keep this an "optional" dependency (same way we deal with pandas.)